.github/workflows/pr.yml

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 # This is the main workflow that runs on every PR and push to main
-name: pr
+name: pull_request
```

Comment:
I vaguely recall having problems with using underscores in these strings. I think it would break yq/jq. Perhaps not a problem now that you've switched to a Python script.